docs/AboutUs.md

```diff
@@ -26,7 +26,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/chewypiano.png" width="200px">
 
 [github](http://github.com/chewypiano)
-[portfolio](team/kamil.md)
+[portfolio](team/chewypiano)
```

Comment:
Missing .md at the back for some of them

Reply:
this is the correct format. Just checked on my fork and this gives the correct link